hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HStore.java

```diff
@@ -431,7 +437,7 @@ public static long determineTTLFromFamily(final ColumnFamilyDescriptor family) {
     return ttl;
   }
 
-  StoreContext getStoreContext() {
+  public StoreContext getStoreContext() {
```

Comment:
Crypto context is table-scoped? We have to get it from the Store?

Reply:
Honestly, I can't remember now why I had duplicated `cryptoContext` on `HStore` here, when it's already defined on `StoreContext`. It doesn't seem correct to me now, let me remove. The need to expose `getStoreContext` as `public` stands, though, as we need that from the `DirectStoreCompactor.createWriterInFamilyDir`, which is in a separate package.